draft-yasskin-http-origin-signed-responses.md

```diff
@@ -386,6 +386,58 @@ extended diagnostic notation from {{?I-D.ietf-cbor-cddl}} appendix G:
 ]
 ~~~
 
+## Loading a certificate chain ## {#cert-chain}
+
+The resource at a signature's `certUrl` MUST have the
+`application/cert-chain+cbor` content type, MUST be canonically-encoded CBOR
+({{canonical-cbor}}), and MUST match the following CDDL:
+
+~~~cddl
+cert-chain = [
+  "📜⛓", ; U+1F4DC U+26D3
+  + {
+    cert: bytes,
+    ? ocsp: bytes,
+    ? sct: bytes,
```

Comment:
A `SignedCertificateTimestampList` appears to be a serialized TLS message, but I guess because other systems have to generate this, it's ok?

Reply:
Right, this is passed as (effectively) opaque input, much like the OCSP response, into the certificate validation system, which contains a minimal subset of the TLS decoder for this case.